src/app/api/routes/users.py

```diff
@@ -26,6 +27,14 @@ async def update_my_info(payload: UserInfo, me: UserRead = Security(get_current_
     """
     Updates information of the current user
     """
+    # Check for access login
+    if payload.login is not None:
+        updated_acccess = await crud.fetch_one(accesses, {"login": me.login})
+        await check_for_access_login_existence(payload.login)
+        if updated_acccess["login"] != payload.login:
+            await check_for_access_login_existence(payload.login)
```

Comment:
Perhaps I'm not following, but we don't need to run `await check_for_access_login_existence(payload.login)
` twice (the first one can be removed I think)

Reply:
Oups good catch